### PR TITLE
Fix 9779 mutations api endpoint oncotator fields

### LINKED ISSUE #9779: Remove dbSnp* and Oncotator* related fields in frontend and backend
Remove OncotatorProteinPosStar, OncotatorProteinPosEnd, getOncotatorRefseqMrnaId fields in frontend and backend

The changes require both frontend and backend changes. Frontend can be release ahead of backend or both releases have to happen in sync.

OncotatorProteinPosStart
OncotatorProteinPosEnd
refseqMrnaId - Has some frontend code as well

 Removing oncotator fields

Backend changes:
- [x] web/src/main/java/org/cbioportal/web/mixin/MutationMixin.java
- [x] web/src/main/java/org/cbioportal/web/parameter/sort/MutationSortBy.java

- [x] core/src/test/java/org/mskcc/cbio/portal/dao/TestDaoMutation.java

- [x] core/src/main/java/org/mskcc/cbio/maf/MafRecord.java
- [x] 	core/src/main/java/org/mskcc/cbio/maf/MafUtil.java
- [x] 	core/src/main/java/org/mskcc/cbio/portal/dao/DaoMutation.java
- [x] 	core/src/main/java/org/mskcc/cbio/portal/model/ExtendedMutation.java
- [x] core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java
- [x] core/src/main/java/org/mskcc/cbio/portal/util/ExtendedMutationUtil.java
- [x] model/src/main/java/org/cbioportal/model/Mutation.java
- [x] persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/MutationMyBatisRepositoryTest.java
- [x] persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/MutationMapper.xml

SQL

- [x]   cgds.sql
- [x]   cgds-test.sql
- [x]   testSql.sql
- [x]   core/src/test/resources/seed_mini.sql

### PR DESCRIPTION
**Pull request by jagnathan**
Fix #9779 
- Removing ONCOTATOR_DBSNP_RS and ONCOTATOR_UNIPROT_ENTRY_NAME  from mutation_event table
- Backend code changes for removal of above fields

### CODE REVIEW

File: core/src/main/java/org/mskcc/cbio/maf/MafUtil.java [hunk 1]
```diff
@@ -116,10 +116,8 @@ public class MafUtil {
     public static final String PROTEIN_POSITION = "Protein_position";
 
     // oncotator column names
-    public static final String ONCOTATOR_DBSNP_RS = "ONCOTATOR_DBSNP_RS";
     public static final String ONCOTATOR_VARIANT_CLASSIFICATION = "ONCOTATOR_VARIANT_CLASSIFICATION";
     public static final String ONCOTATOR_REFSEQ_MRNA_ID = "ONCOTATOR_REFSEQ_MRNA_ID";
-    public static final String ONCOTATOR_UNIPROT_ENTRY_NAME = "ONCOTATOR_UNIPROT_ENTRY_NAME";
     public static final String ONCOTATOR_UNIPROT_ACCESSION = "ONCOTATOR_UNIPROT_ACCESSION";
     public static final String ONCOTATOR_CODON_CHANGE = "ONCOTATOR_CODON_CHANGE";
     public static final String ONCOTATOR_PROTEIN_POS_START = "ONCOTATOR_PROTEIN_POS_START";
```
Comment (inodb) [inline core/src/main/java/org/mskcc/cbio/maf/MafUtil.java:123]: seems like all oncotoator_x fields can be removed?

Comment (inodb) [reply core/src/main/java/org/mskcc/cbio/maf/MafUtil.java:123]: also below all mutation assessor references can be removed (could be separate PR but just fyi)

Comment (jagnathan) [reply core/src/main/java/org/mskcc/cbio/maf/MafUtil.java:123]: mutation assessor references are removed. the public DB has values in the oncotator_xxx columns which have been renamed.

**Comment by dippindots** [comment]
This is included in the `v5.3.0`, we can close this now.

**Pull request closed without merging**